.travis.yml

```diff
@@ -28,7 +28,7 @@ install:
 - pip install tox ${CODECOV:+codecov}
 - if [ ${WITH_RUST:-true} != "false" ]; then curl https://sh.rustup.rs | sh -s -- -y || travis_terminate 1; fi
 - export PATH=$PATH:$HOME/.cargo/bin
-- export AWS_SHARED_CREDENTIALS_FILE=./automock/credentials.cfg
+- export AWS_SHARED_CREDENTIALS_FILE=./automock/boto.cfg
```

Comment:
is this still necessary? (boto.cfg should probably be killed at this point if anything)

Reply:
It is, because rusoto wants to find itself some aws access/secret keys, and if it can't find them in a file it expects, it starts issuing HTTP calls looking for IAM keys and such which messes things up. This is also why I added random access/secret keys to the boto.cfg.

Reply:
What credentials.cfg? https://github.com/mozilla-services/autopush/tree/master/automock